### DISCUSSION TITLE
Can't loop one track?

### DISCUSSION THREAD

**Discussion by bluemangoop** [audacious-media-player/audacious#66] [General]
I like that there's a repeat/loop button, but you should be able to play one song on repeat. 

Aside from that, I love the program.

**Comment by jlindgren90** [comment]
Maybe it's not very intuitive but you can do this already: check both "No Playlist Advance" and "Repeat".

**Reply by bluemangoop** [reply]
Gotcha. And that was a really fast response haha
Thanks.
